src/log/log_auto.c

```diff
@@ -369,10 +380,12 @@ __wt_logop_row_remove_print(
 	WT_RET(__wt_logop_row_remove_unpack(
 	    session, pp, end, &fileid, &key));
 
-	fprintf(out, " \"optype\": \"row_remove\",\n");
-	fprintf(out, "        \"fileid\": \"%" PRIu32 "\",\n", fileid);
+	WT_RET(__wt_fprintf(out, " \"optype\": \"row_remove\",\n"));
+	WT_RET(__wt_fprintf(out,
+	    "        \"fileid\": \"%" PRIu32 "\",\n", fileid));
 	WT_RET(__logrec_jsonify_str(session, &escaped, &key));
-	fprintf(out, "        \"key\": \"%s\"", escaped);
+	WT_RET(__wt_fprintf(out,
+	    "        \"key\": \"%s\"", escaped));
 	__wt_free(session, escaped);
```

Comment:
@keithbostic, likewise there's a few more of these `escaped` that need to be freed on error.